sys/include/net/gnrc/sixlowpan/config.h

```diff
@@ -81,12 +81,29 @@ extern "C" {
 #endif
 
 /**
- * @brief   Timeout for reassembly buffer entries in microseconds
+ * @brief   Timeout for reassembly buffer entries in milliseconds
  *
  * @note    Only applicable with
  *          [gnrc_sixlowpan_frag_rb](@ref net_gnrc_sixlowpan_frag_rb) module
  */
+#ifndef CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_MS
 #ifndef CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US
+#define CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_MS (3U)
+#else
+#define CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_MS (CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US / US_PER_SEC)
+#endif
+#endif
+
+/**
+ * @brief   Timeout for reassembly buffer entries in microseconds
+ *
+ * @note    Only applicable with
+ *          [gnrc_sixlowpan_frag_rb](@ref net_gnrc_sixlowpan_frag_rb) module
+ *
+ * @deprecated Use @ref CONFIG_GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_MS instead.
+ *          Will be removed after 2023.01 release.
```

Comment:
The usual deprecation cycle is two releases, since the 2023.01 release is just up ahead, I think, 2023.07 is the better release to deprecate. This way users still have some time to adjust ;-).

Reply:
Ping?